lib/sourcemapper.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/** @const */ var _ = require('lodash');
+/** @const */ var async = require('async');
+/** @const */ var fs = require('fs');
+/** @const */ var path = require('path');
+
+/** @const */ var sourceMap = require('source-map');
+
+/** @define {string} */ var MAP_EXT = '.map';
+
+function create(sourcemapPaths, callback) {
+  var mapper = new SourceMapper();
+  var callList = Array.prototype.slice.call(sourcemapPaths)
+                 .map(function(path) {
+                   return function(callback) {
+                     processSourcemap(mapper.infoMap_, path, callback);
+                   };
+                 });
+
+  async.parallel(callList, function(err) {
```

Comment:
Perhaps use `async.parallelLimit` to ensure we don't IO-bomb the app startup if there are too many source map paths? I don't think there is a significant urgency in our module to finish the source map process ASAP.